managed-ledger/src/main/java/org/apache/bookkeeper/mledger/LedgerOffloader.java

```diff
@@ -115,6 +217,16 @@
     CompletableFuture<Void> deleteOffloaded(long ledgerId, UUID uid,
                                             Map<String, String> offloadDriverMetadata);
 
+    default CompletableFuture<ReadHandle> readOffloaded(long ledgerId, MLDataFormats.OffloadContext ledgerContext,
+                                                        Map<String, String> offloadDriverMetadata) {
+        throw new UnsupportedClassVersionError();
```

Comment:
UnsupportedOperationException?

Reply:
It's the default behavior of the interface, useful when a new implementation only implements a part of the interface but still useful.